seqr/utils/search/elasticsearch/es_utils_tests.py

```diff
@@ -11,9 +11,9 @@
 from urllib3.exceptions import ReadTimeoutError
 
 from seqr.models import Family, Sample, VariantSearch, VariantSearchResults
-from seqr.utils.elasticsearch.utils import get_es_variants_for_variant_tuples, get_single_es_variant, get_es_variants, \
+from seqr.utils.search.utils import get_es_variants_for_variant_tuples, get_single_es_variant, get_es_variants, \
     get_es_variant_gene_counts, get_es_variants_for_variant_ids, InvalidIndexException, InvalidSearchException
-from seqr.utils.elasticsearch.es_search import _get_family_affected_status, _liftover_grch38_to_grch37
+from seqr.utils.search.elasticsearch.es_search import _get_family_affected_status, _liftover_grch38_to_grch37
```

Comment:
I suggest removing the first underscore for the public function names.

Reply:
this PR is only moving files, not changing anything else. Subsequent PRs can clean up functions calls but the diff would not be readable if I start changing things in the files